akka-http-core/src/main/scala/akka/http/impl/engine/ws/WebSocket.scala

```diff
@@ -40,6 +40,7 @@ private[http] object WebSocket {
     closeTimeout:      FiniteDuration    = 3.seconds, // TODO put close timeout into the settings?
     log:               LoggingAdapter): BidiFlow[FrameEvent, Message, Message, FrameEvent, NotUsed] =
     masking(serverSide, websocketSettings.randomFactory) atop
+      FrameLogger.logFramesIfEnabled(websocketSettings.logFrames) atop
```

Comment:
To fit here, the type had to be adapted to accept `FrameEventOrError`. The benefit is that frames are shown unmasked.